lusSTR/wrappers/convert.py

```diff
@@ -58,16 +58,25 @@ def format_table(input, software, kit="forenseq", custom=False):
             locus = "PENTA E"
         if locus == "DYS385A/B" or locus == "DYS385":
             locus = "DYS385A-B"
-        if locus == "AMELOGENIN":
-            continue
         metadata = str_marker_data[locus]
         if kit == "forenseq":
             remove_5p = metadata["Foren_5"]
             remove_3p = metadata["Foren_3"]
         else:
             remove_5p = metadata["Power_5"]
             remove_3p = metadata["Power_3"]
-        if len(sequence) <= (remove_5p + remove_3p) and software != "uas":
+        if custom:
+            if metadata["Custom_5"] < 0:
+                remove_5p = remove_5p - metadata["Custom_5"]
+            if metadata["Custom_3"] < 0:
+                remove_3p = remove_3p - metadata["Custom_3"]
+        if (
+            len(sequence) <= (remove_5p + remove_3p + len(metadata["LUS"]))
+            and software != "uas"
+            and locus != "AMELOGENIN"
+        ) or (
+            software != "uas" and locus == "AMELOGENIN" and len(sequence) < (remove_5p + remove_3p)
+        ):
```

Comment:
This conditional is dizzying!

Reply:
I know, it's complicated. The Amelogenin true X chr sequences were getting thrown out because it's 0 bases after removal... but that first statement is still necessary for all the other markers.

Reply:
I don't doubt it's necessary. I'll see if I can come up with a way to make the syntax a little less inscrutable.

Reply:
How about:
```
flanks_removed_len = remove_5p + remove_3p
amel_remove = len(sequence) < flanks_removed_len
otherloci_remove = len(sequence) <= (flanks_removed_len + len(metadata["LUS"]))
if (
      software != "uas"
      and locus != "AMELOGENIN"
      and otherloci_remove
) or (
      software != "uas" and locus == "AMELOGENIN" and amel_remove
):
```

Reply:
It ended up being ```-=1``` but is updated and working!